src/theme/styles/colours.ts

```diff
@@ -112,13 +116,15 @@ export const DARK_COLOURS = {
   textActive1: '#D96D49',
   textDisabled: '#31323E',
 
-  icon: '#8D8DA980',
+  icon: '#8D8DA9',
```

Comment:
Opting here for using a single color for the icon (without Alpha). The using `opacity` to set on hover effects.